### DISCUSSION TITLE
Problems since the MSFS update

### DISCUSSION THREAD

**Discussion by CHAussie** [nguyenquyhy/Flight-Tracker-StreamDeck#182] [General]
Hey there

since the update I have the problem that the Flight Tracker connection seems to be very unstable. Sometimes there is a lag with the input, if you release a key the command is still sent to the sim. Often some buttons get inresponsive. My programmed keys for parking brake and gear display the correct state (on/off) but can no longer be switched with the button.

Is it possible that we need an update of the plugin?

Oliver

**Comment by nguyenquyhy** [comment]
Weird. I just tested and it works quite normally. Any specific aircraft you are testing on?
